packages/lit-dev-content/site/tutorials/content/working-with-lists/02.md

```diff
@@ -0,0 +1,48 @@
+Any valid JavaScript expressions that result in valid renderable values
+described in the [expressions documentation](/docs/templates/expressions/) can
+be placed within the tagged template literal. When working with arrays, their
```

Comment:
so the expressions have to be placed in template literal _expressions_. Saying that would be three "expressions" in one sentence though. Maybe the sentence could be reordered:

Lit template literal expressions can contain any JavaScript that returns a renderable value as described in...